core/init.go

```diff
@@ -0,0 +1,108 @@
+package core
+
+import (
+	"fmt"
+	"path/filepath"
+
+	tmrand "github.com/celestiaorg/celestia-core/libs/rand"
+	"github.com/celestiaorg/celestia-core/p2p"
+	"github.com/celestiaorg/celestia-core/privval"
+	"github.com/celestiaorg/celestia-core/types"
+	tmtime "github.com/celestiaorg/celestia-core/types/time"
+
+	"github.com/celestiaorg/celestia-node/libs/utils"
+)
+
+// defaultValKeyType is default crypto algo for consensus in core
+// TODO(@Wondertan): Should we make this configurable
+const defaultValKeyType = types.ABCIPubKeyTypeSecp256k1
```

Comment:
No, the configuration of which key type is used during consensus is none of the business of the celestia-node. It should not be configurable here. Remember, we are treating tendermint as a black-box.

Reply:
Ok, removing the todo.

Reply:
Resolved?